cloudera/datadog_checks/cloudera/check.py

```diff
@@ -0,0 +1,95 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import datetime
+import time
+
+from datadog_checks.base import AgentCheck
+
+from .client_factory import make_api_client
+from .common import API_ENTITY_STATUS, CAN_CONNECT, CLUSTER_HEALTH, CLUSTER_RESOURCE_API, SERVICE_HEALTH, SERVICE_RESOURCE_API
+from .queries import TIMESERIES_QUERIES
+
+
+class ClouderaCheck(AgentCheck):
+    __NAMESPACE__ = 'cloudera'
+
+    def __init__(self, name, init_config, instances):
+        super(ClouderaCheck, self).__init__(name, init_config, instances)
+
+        self.client, self._error = make_api_client(self, self.instance)
+        if self.client is None:
+            self.error("API Client is none: %s", self._error)
+            self.service_check(CAN_CONNECT, AgentCheck.CRITICAL)
+
+        self.custom_tags = self.instance.get("tags", [])
+
+    def run_timeseries_checks(self):
+        to_time = datetime.datetime.fromtimestamp(time.time())
+        from_time = datetime.datetime.fromtimestamp(time.time() - 6000)
```

Comment:
time() returns seconds, why are we subtracting 6000?

Reply:
I would prefer something like:
```
now = time.time()
to_time = datetime.datetime.fromtimestamp(now)
from_time = datetime.datetime.fromtimestamp(now - 60)
```

Reply:
I ended up removing figuring out the timestamp, since by default the `to_time` is already set to `now`, `from_time` is by default 5 minutes before now, and we always only get the most recent entry of the queries.